### PR TITLE
SCO: Update UI based on UX team feedback #19965

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
As a developer, I need to update the SCO page UI based on UX feedback so that it meets DSVA standards and design practices.

https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19965

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: src/site/components/merger-social-sco.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
+{% comment %}
+======================
+Social Media - R. Rail
+======================
+
+See https://help.shopify.com/themes/liquid/tags/theme-tags#include
```
Comment (cvalarida) [inline src/site/components/merger-social-sco.html]: I'm not sure I understand why this is here.

File: src/site/components/merger-social-sco.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
+{% comment %}
+======================
+Social Media - R. Rail
```
Comment (cvalarida) [inline src/site/components/merger-social-sco.html]: What's this?

File: src/site/includes/education-sco.html [hunk 3]
```diff
@@ -48,7 +48,7 @@ <h1>
         {% endif %}
 
         {% if social != empty %}
-          {% include "src/site/components/merger-social.html" %}
+          {% include "src/site/components/merger-social-sco.html" %}
```
Comment (rianfowler) [inline src/site/includes/education-sco.html]: @ncksllvn just an fyi- i'm not sure why we wouldn't want this to be applied across the site if it better meets the design standards.

**Review by rianfowler — APPROVED** [review]

File: src/site/includes/education-sco.html (hunk 3 above)
Comment (zurbergram) [reply src/site/includes/education-sco.html]: https://github.com/department-of-veterans-affairs/vets-website/blob/05e109dcf005b9c30e47dad1cd13617209185472/src/site/components/merger-social-sco.html#L45
https://github.com/department-of-veterans-affairs/vets-website/blob/05e109dcf005b9c30e47dad1cd13617209185472/src/site/components/merger-social-sco.html#L50

These 2 lines are specific to https://github.com/department-of-veterans-affairs/vagov-content/blob/master/pages/education/school-resources.md

File: src/site/components/merger-social-sco.html (hunk 1 above)
Comment (zurbergram) [reply src/site/components/merger-social-sco.html]: This file is a copy and past of https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/components/merger-social.html

File: src/site/components/merger-social-sco.html (hunk 2 above)
Comment (zurbergram) [reply src/site/components/merger-social-sco.html]: This file is a copy and past of https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/components/merger-social.html

File: src/site/includes/education-sco.html (hunk 3 above)
Comment (rianfowler) [reply src/site/includes/education-sco.html]: thanks @zurbergram

**Pull request merged**